cnxarchive/tests/testing.ini

```diff
@@ -4,7 +4,7 @@ db-connection-string = dbname=cnxarchive-testing user=cnxarchive password=cnxarc
 memcache-servers = localhost
 search-cache-expiration = 15
 search-long-cache-expiration = 30
-
+sitemap-destination = http://cnx.org/sitemap.xml
```

Comment:
This looks like a required setting that isn't going to fail at startup. So let's make one minor change. Please add this setting to the `manditory_settings` list in https://github.com/Connexions/cnx-archive/blob/f4522c5087c7a44d6e8726dc91a2adf46ac5a5c9/cnxarchive/__init__.py#L81